cpp/include/cudf/binaryop.hpp

```diff
@@ -214,5 +214,47 @@ cudf::data_type binary_operation_fixed_point_output_type(binary_operator op,
                                                          cudf::data_type const& lhs,
                                                          cudf::data_type const& rhs);
 
+namespace binops {
+
+/**
+ * @brief Computes output valid mask for op between a column and a scalar
+ *
+ * @param col     Column to compute the valid mask from
+ * @param s       Scalar to compute the valid mask from
+ * @param stream  CUDA stream used for device memory operations and kernel launches
+ * @param mr      Device memory resource used to allocate the returned valid mask
+ * @return        Computed validity mask
+ */
+rmm::device_buffer scalar_col_bitmask_and(
```

Comment:
In https://github.com/rapidsai/cudf/pull/9616, we refactored `bitmask_and` to return a pair of the bitmask and the null count. This function should do the same. It has all the needed information to return the null count, so callers of this function shouldn't need to use `cudf::UNKNOWN_NULL_COUNT`.